.github/workflows/ansible-test.yml

```diff
@@ -1,5 +1,5 @@
 name: CI
-on: push
+on: pull_request
```

Comment:
we should trigger action on both push and pull requests : 
```
on: [push, pull_request]
```

Reply:
https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#on